src/lib/Components/Bidding/Elements/Typography.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import styled from "styled-components/native"
+import { color, fontSize, lineHeight, space, textAlign } from "styled-system"
+
+import { Fonts } from "../../../data/fonts"
+
+const Serif = styled.Text`
+  font-family: "${Fonts.GaramondRegular}";
+  ${color}
+  ${fontSize}
+  ${lineHeight}
+  ${space}
+  ${textAlign}
+`
+
+const SerifItalic = styled.Text`
+  font-family: "${Fonts.GaramondItalic}";
+  ${color}
+  ${fontSize}
+  ${lineHeight}
+  ${space}
+  ${textAlign}
+`
+
+export const Serif12 = props => <Serif fontSize={1} {...props} />
+export const Serif14 = props => <Serif fontSize={2} {...props} />
+export const Serif16 = props => <Serif fontSize={3} {...props} />
+export const Serif18 = props => <Serif fontSize={4} {...props} />
```

Comment:
Shouldn't the naming convention here be based on the "modifier" that's part of the type system?

I.e. looking at the chart below, I would expect `Serif12` to have a font size not of 12, but of 66

![font naming](https://user-images.githubusercontent.com/140521/39066981-0f67a9e8-44a5-11e8-8c96-f82bac308d0a.png)

Reply:
Using the modifiers as the naming convention totally makes sense. I'll update the PR.

Reply:
@anandaroop I just looked to see if we could use a `modifier` or a relative scale as an interface, but we've actually discussed whether or not we should be using it at the meeting on Monday. One concern we discussed was that when we had to add/remove one `px` size to/from the type (as in typography) system, this would require a breaking change if the scale is relative. So I named them following the naming convention of `${typeface}${size}`. I'm not opinionated about this that much, but breaking changes will happen for sure.

cc @acjay @damassi @l2succes @briansw

Reply:
Thanks for thinking through this @yuki24 !

I think it's a fair point to wonder about breaking changes. I don't have a good intuition about that, because I don't know what degree of finality to expect from this (or any) design system.

As for relying on `${typeface}${size}` as the naming convention, note that this may not provide us with unique names. Consider `Serif16` — does that refer to 16/24 or 16/18? (In the chart above 16/18 is one of the **tightly** linespaced variants indicated by the **"t"** in the modifier). 

(I think the decision to abstract style names from their css details is common to "design system" approaches to css — I'm thinking of examples like [Tachyon's type scale](http://tachyons.io/docs/typography/scale/) or even [BassCSS](http://basscss.com/#basscss-type-scale) which is an immediate inspiration for `styled-system`. I think there may be some hard won wisdom there, but I personally am not super opinionated about the naming choice here either.)